src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -1839,14 +1840,23 @@ TEST_CASE(
     params.async = true;
     delays = {100};
   }
-  SUBCASE(
-      "async - slow response with sequences on should slow down our send rate")
+  SUBCASE("async - slow response with sequences on")
   {
     is_sequence_model = true;
     params.async = true;
     params.num_of_sequences = 5;
     delays = {100};
-    expected_count = params.num_of_sequences;
+
+    SUBCASE("single live request per sequence should slow down our send rate")
```

Comment:
This text doesn't make sense.

Reply:
unless "single_live_request_per_sequence" is the name of the new option.

Reply:
I pulled that text from the proposal doc.
I agree that we need to cement a name for the feature soon.

Reply:
fix subcase names.